project/SparkBuild.scala

```diff
@@ -1208,6 +1254,9 @@ object CopyDependencies {
           if (jar.getName.contains("spark-connect") &&
             !SbtPomKeys.profiles.value.contains("noshade-connect")) {
             Files.copy(fid.toPath, destJar.toPath)
+          } else if (jar.getName.contains("spark-protobuf") &&
+            !SbtPomKeys.profiles.value.contains("noshade-protobuf")) {
+            Files.copy(fid.toPath, destJar.toPath)
```

Comment:
You have the variable `fidProtobuf`, but you don't use it anywhere. Possibly it should be used here?

Reply:
@bersprockets thanks for pointing this out; I have made changes in the follow-up [PR](https://github.com/apache/spark/pull/38212).